### DISCUSSION TITLE
[Enhancement]: Show genre for album on album view

### DISCUSSION THREAD

**Discussion by lukearnould** [ciderapp/Cider#657] [Added Features]
### The feature you would like

I noticed that the genre for an album is not visible on the page, whether viewing from search or within the library. The genre for a song can be viewed in the Songs sidebar item, so it seems like this value is at least visible in the MusicKit API.

While Apple Music's genre implementation and categories are admittedly not great, they can be very useful when telling apart newly discovered albums and quickly being able to tell if I might like the album.

If trying to follow the iOS Apple Music app's presentation of genres while being consistent with Cider's current design, the genre could be displayed next to the release date at the bottom of the album page. Though I must admit that showing it near the top, such as in an area beneath the album artist, would be much faster to view.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by cryptofyre** [comment]
Added via #790 

Locking discussion.
